components/chart/tooltip/overview.md

```diff
@@ -0,0 +1,224 @@
+
+* `Color` - control the text color by applying a CSS color string, including HEX and RGB.
+
+* `Border` - control the `Color` and the `Width` of the tooltip by using the `<ChartSeriesTooltipBorder />` nested inside the `<ChartSeriesTooltip>` tag.
+
+* `Padding` - control the `Left`, `Right`, `Top` and `Bottom` padding of the tooltip by using the `<ChartSeriesTooltipPadding />` nested inside the `<ChartSeriesTooltip>` tag.
+
+* Use the [Template](#template) to take control over what is rendered in the tooltip.
+
+>caption Configuration of the tooltips with applied customization settings
+
+````CSHTML
+@* This example shows how to customize the tooltip *@
+
+<TelerikChart>
+    <ChartSeriesItems>
+        <ChartSeries Type="ChartSeriesType.Line" Name="Product 1 (bound to simple data)" Data="@simpleData">
+        </ChartSeries>
+        <ChartSeries Type="ChartSeriesType.Line" Name="Product 2 (bound to model)"
+                     Data="@modelData" Field="@nameof(MyDataModel.SecondSeriesValue)">
+            <ChartSeriesTooltip Visible="true"
+                                Background="#0000FF"
+                                Color="#D3D3D3">
+                <ChartSeriesTooltipBorder Color="#ADD8E6" Width="10" />
+                <ChartSeriesTooltipPadding Top="10" Bottom="10" Left="15" Right="15" />
+            </ChartSeriesTooltip>
+        </ChartSeries>
+    </ChartSeriesItems>
+
+    <ChartValueAxes>
+        <ChartValueAxis Color="red"></ChartValueAxis>
+    </ChartValueAxes>
+
+    <ChartCategoryAxes>
+        <ChartCategoryAxis Categories="@xAxisItems"></ChartCategoryAxis>
+    </ChartCategoryAxes>
+
+    <ChartTitle Text="Quarterly sales trend"></ChartTitle>
+
+    <ChartLegend Position="Telerik.Blazor.ChartLegendPosition.Bottom">
+    </ChartLegend>
+</TelerikChart>
+
+@code {
+    public class MyDataModel
+    {
+        public int SecondSeriesValue { get; set; }
+        public string ExtraData { get; set; }
+
+    }
+
+    public List<MyDataModel> modelData = new List<MyDataModel>()
+    {
+        new MyDataModel() { SecondSeriesValue = 1, ExtraData = "first" },
+        new MyDataModel() { SecondSeriesValue = 5, ExtraData = "second" },
+        new MyDataModel() { SecondSeriesValue = 3, ExtraData = "third" },
+        new MyDataModel() { SecondSeriesValue = 2, ExtraData = "fourth" },
+    };
+
+    public List<object> simpleData = new List<object>() { 10, 2, 7, 5 };
+
+    public string[] xAxisItems = new string[] { "Q1", "Q2", "Q3", "Q4" };
+}
+
+````
+
+>caption The result from the code snippet above
+
+![tooltip parameter customization example](images/tooltip-customized-example.png)
+
+### Template
+
+The `Template` allows you to take control over the rendering of the tooltip and include additional information to the user.
+
+It provides a `context` parameter with the following information:
+
+* `Value` - maps to the value of the data point that is being hovered.
```

Comment:
Since these are also provided as child properties of the Points in the shared template, consider extracting this list to a reusable template to make all the information available in both articles without copying